core/exception/case_compare_spec.rb

```diff
@@ -1,5 +1,35 @@
 require_relative '../../spec_helper'
 
 describe "SystemCallError.===" do
-  it "needs to be reviewed for spec completeness"
+  before :all do
+    @example_errno = 22
```

Comment:
Could you use `Errno::EINVAL::Errno` here? That way, we don't hardcode any number.

Reply:
Wonderful!  I tried a different strategy (see below) that didn't work.